motion/p5js/connected-elements/index.js

```diff
@@ -0,0 +1,20 @@
+function setup() {
+  createCanvas(500,500);
```

Comment:
Another small change. The canvas needs to be assigned to a specific parent element:  

```
  var canvas = createCanvas(500, 500); // !this is important
  canvas.parent('sketch'); // !this as well
```